pkg/policy/l4.go

```diff
@@ -1201,7 +1201,11 @@ func (sp *PerSelectorPolicy) redirectType() redirectTypes {
 func (l4 *L4Filter) Marshal() string {
 	b, err := json.Marshal(l4)
 	if err != nil {
-		b = []byte("\"L4Filter error: " + err.Error() + "\"")
+		jsonErr, err2 := json.Marshal(err.Error())
+		if err2 != nil {
+			jsonErr = []byte("unable to marshall error")
+		}
+		return "L4Filter error: \"" + string(jsonErr) + "\""
```

Comment:
JSON marshal on a string will anyways encode the error in double quotes, we don't need to do it again when returning the string.

```suggestion
		return "L4Filter error: " + string(jsonErr)
```

Reply:
Ah right, nice catch. Fixed.